auto3dseg/algorithm_templates/dints/scripts/train.py

```diff
@@ -318,7 +363,9 @@ def run(config_file: Optional[Union[str, Sequence[str]]] = None, **override):
                         val_labels = post_label(val_labels[0, ...])
                         val_labels = val_labels[None, ...]
 
-                    value = compute_meandice(y_pred=val_outputs, y=val_labels, include_background=not softmax)
+                    value = compute_meandice(
```

Comment:
Function `compute_meandice` has been deprecated since version 1.0.0. Can we change to use `compute_dice` instead? Thanks!